Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -1102,6 +1115,19 @@ namespace AzToolsFramework
         }
     }
 
+    void PropertyRowWidget::CreateGroupToggleSwitch()
+    {
+        if (!m_toggleSwitch)
+        {
+            m_handlerName = AZ::Edit::UIHandlers::CheckBox;
+            EBUS_EVENT_RESULT(m_handler, PropertyTypeRegistrationMessages::Bus, ResolvePropertyHandler, m_handlerName, azrtti_typeid<bool>());
```

Comment:
update: We're trying to move away from the old EBus macro calling convention, would you be able to please use the standard template approach instead. I think it should be...

```
PropertyTypeRegistrationMessages::Bus::EventResult(
    m_handler, &PropertyTypeRegistrationMessages::Bus::Events::ResolvePropertyHandler,
    m_handlerName, azrtti_typeid<bool>());
```

Reply:
Sounds good, I've put in the new EBUS template.

Reply:
Thanks!